docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -1704,7 +1704,7 @@ How it **works**:
 Common **causes**:
 
 - Your code has a single target that exposes the same time series multiple times, or multiple targets with identical labels.
-- System time of your Prometheus instance has been shifted backwards. If this was a mistake, fix the system time back to normal. Otherwise, wait until the system time catches up to the time it was changed.
+- System time of your Prometheus instance has been shifted backwards. If this was a mistake, fix the system time back to normal. Otherwise, wait until the system time catches up to the time it was changed. You can run this query to identify clock skew on any hosts `abs(node_time_seconds - timestamp(node_time_seconds))`
```

Comment:
```suggestion
- System time of your Prometheus instance has been shifted backwards. If this was a mistake, fix the system time back to normal. Otherwise, wait until the system time catches up to the time it was changed. Using [node exporter](https://github.com/prometheus/node_exporter) metrics identify clock skew on any hosts using this promQL query: `abs(node_time_seconds - timestamp(node_time_seconds))`.
```